watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/Users/RegisterUserRequest.java

```diff
@@ -0,0 +1,17 @@
+package com.blueprint.watershed.Networking.Users;
+
+import android.app.Activity;
+
+import com.android.volley.Response;
+import com.blueprint.watershed.Users.User;
+
+import org.json.JSONObject;
+
+/**
+ * Created by charlesx on 4/13/15.
+ */
+public class RegisterUserRequest extends UpdateUserRequest {
+    public RegisterUserRequest(final Activity activity, User user, JSONObject params, final Response.Listener<User> listener) {
+        super(activity, user, params, listener, makeUserResourceURL(user.getId(), "register"));
```

Comment:
Seems like there's no reason to have this class. You're literally passing all of the parameters directly to super. Why not just call that with the desired url in the first place?